EIPS/eip-190-smart-contract-packaging-standard.md

```diff
@@ -0,0 +1,102 @@
+```
+EIP: Draft
+Title: Ethereum Smart Contract Packaging Standard
+Authors: Piper Merriam, Tim Coulter, Denis Erfurt (mhhf), RJ Catalano (VoR0220), Iuri Matias (iurimatias)
+Status: Draft
+Type: Standards Track - ERC
+Created: 2017-01-10
+```
+
+# Abstract
+
+This ERC proposes a specification for Ethereum smart contract packages.  
+
+The specification was collaboratively developed by the following Ethereum development framework maintainers.
+
+* Tim Coulter (Truffle)
+* Denis Erfurt (Dapple)
+* Piper Merriam (Populus)
+* RJ Catalano (Eris PM)
+* Iuri Matias (Embark)
+
+# Motivation
+
+Packaging is a core piece of modern software development which is missing from the Ethereum ecosystem.  The lack of packaging limits the ability for developers to reuse code which negatively affects productivity and security.
+
+A key example of this is the ERC20 standard.  There are a few well audited reusable token contracts available but most developers end up writing their own because of the difficulty in finding and reusing existing code.
+
+A packaging standard should have the following positive effects on the ecosystem:
+
+* Greater overall productivity caused by the ability to reuse existing code.
+* Increased security caused by the ability to reuse existing well audited implementations of common patterns (ERC20, crowdfunding, etc).
+
+Smart contract packaging should also have a direct positive effect on the end user.  Wallet software will be able to consume a released package and generate an interface for interacting with any deployed contracts included within that package.  With the advent of [ENS](https://github.com/ethereum/EIPs/issues/137) all of the pieces will be in place for a wallet to take a human readable name and present the user with an interface for interacting with the underlying application.
+
+
+# Specification
+
+The full specification for this standard is maintained separately in the repository [epm/epm-spec](https://github.com/ethpm/epm-spec).
+
+This EIP refers to the `1.0.0` version of the specification: [https://github.com/ethpm/epm-spec/tree/v1.0.0](https://github.com/ethpm/epm-spec/tree/v1.0.0) 
+
+The specification contains details for a single document referred to as a *"Release Lockfile"*.  
+
+* Release Lockfile Specification: [https://github.com/ethpm/epm-spec/blob/v1.0.0/release-lockfile.spec.md](https://github.com/ethpm/epm-spec/blob/v1.0.0/release-lockfile.spec.md).
+* JSON Schema for Release Lockfile: [https://github.com/ethpm/epm-spec/blob/v1.0.0/spec/release-lockfile.spec.json](https://github.com/ethpm/epm-spec/blob/v1.0.0/spec/release-lockfile.spec.json)
+
+> These documents have not been inlined into this ERC to ensure that there is a single source of truth for the specification.
+
+
+# Rationale
```

Comment:
Go ahead and remove rationale since you explained it sufficiently in Motivation.